mix.exs

```diff
@@ -73,7 +73,8 @@ defmodule DotCom.Mixfile do
       {:address_us, "0.4.3"},
       {:aws, "1.0.4"},
       {:aws_credentials, "0.3.2", optional: true},
-      {:castore, "1.0.11"},
+      {:castore, "1.0.12"},
+      {:cowlib, "2.14.0", override: true},
```

Comment:
technically not sure this override is essential, but I had a heck of a time trying to get mix to obey me with reconciling the cowboy/cowlib requirements.

Reply:
It's usually because some other dependency depends on this one at a different version number.